rule_engine/src/main.cpp

```diff
@@ -197,25 +280,41 @@ namespace
 
 	auto genquery2_destroy(std::list<boost::any>& _rule_arguments, irods::callback&) -> irods::error
 	{
-		log_rule_engine::info(__func__);
+#ifdef IRODS_ENABLE_42X_COMPATIBILITY
+		rodsLog(LOG_DEBUG, __func__);
+#else
+		log_re::trace(__func__);
+#endif // IRODS_ENABLE_42X_COMPATIBILITY
 
 		if (_rule_arguments.size() != 1) {
 			const auto msg =
 				fmt::format("Incorrect number of input arguments: expected 1, received {}", _rule_arguments.size());
-			log_rule_engine::info(msg);
+#ifdef IRODS_ENABLE_42X_COMPATIBILITY
+			rodsLog(LOG_ERROR, msg.c_str());
+#else
+			log_re::error(msg);
+#endif // IRODS_ENABLE_42X_COMPATIBILITY
 			return ERROR(SYS_INVALID_INPUT_PARAM, msg);
 		}
 
 		try {
 			auto iter = std::begin(_rule_arguments);
 			const auto* ctx_handle = boost::any_cast<std::string*>(*iter);
-			log_rule_engine::info("ctx_handle = [{}]", *ctx_handle); // FIXME It's empty in the PREP! :-(
+#ifdef IRODS_ENABLE_42X_COMPATIBILITY
+			rodsLog(LOG_DEBUG, "ctx_handle = [%s]", ctx_handle->c_str()); // FIXME It's empty in the PREP! :-(
```

Comment:
Log level?

Also, is the FIXME captured in an issue somewhere?

Reply:
This may not be real anymore. Will check.

Reply:
Removed comments because PREP works. I was holding it wrong.